debian/rules

```diff
@@ -4,222 +4,11 @@ export DH_VERBOSE=1
-      -Dhibernate=false \
-      -Dldconfig=false \
-      -Dresolve=false \
-      -Defi=true \
-      -Dtpm=true \
-      -Dtpm-pcrindex=12 \
-      -Denvironment-d=false \
-      -Dbinfmt=false \
-      -Dcoredump=false \
-      -Dlogind=true \
-      -Dhostnamed=false \
-      -Dlocaled=false \
-      -Dmachined=false \
-      -Dportabled=false \
-      -Dnetworkd=false \
-      -Dtimedated=false \
-      -Dtimesyncd=false \
-      -Dremote=false \
-      -Dnss-myhostname=false \
-      -Dnss-mymachines=false \
-      -Dnss-resolve=false \
-      -Dnss-systemd=true \
-      -Dfirstboot=false \
-      -Drandomseed=false \
-      -Dbacklight=false \
-      -Dvconsole=false \
-      -Dquotacheck=false \
-      -Dsysusers=true \
-      -Dtmpfiles=true \
-      -Dimportd=false \
-      -Dhwdb=true \
-      -Drfkill=false \
-      -Dman=false \
-      -Dhtml=false \
-      -Dfallback-hostname=ubuntu \
-      -Dsystem-uid-max=999 \
-      -Dsystem-gid-max=999 \
-      -Dadm-group=true \
-      -Dwheel-group=false \
-      -Ddev-kvm-mode=0660 \
-      -Dgroup-render-mode=0660 \
-      -Ddefault-dnssec=no \
-      -Dselinux=true \
-      -Ddefault-kill-user-processes=false \
-      -Ddefault-hierarchy=unified \
-      -Ddefault-locale=C.UTF-8 \
-      -Dnobody-user=nobody \
-      -Dnobody-group=nogroup \
-      -Dbump-proc-sys-fs-nr-open=false \
-      -Dsupport-url=https://www.ubuntu.com/support \
-      -Dseccomp=true \
-      -Dapparmor=true \
-      -Dsmack=false \
-      -Dpolkit=false \
-      -Dima=false \
-      -Dacl=true \
-      -Daudit=true \
-      -Dblkid=true \
-      -Dkmod=true \
-      -Dpam=true \
-      -Dpamconfdir=no \
-      -Dmicrohttpd=false \
-      -Dlibcryptsetup=true \
-      -Dlibcurl=false \
-      -Dlibidn2=true \
-      -Dlibiptc=true \
-      -Dqrencode=false \
-      -Dgcrypt=false \
-      -Dgnutls=false \
-      -Dopenssl=false \
-      -Delfutils=false \
-      -Dzlib=false \
-      -Dbzip2=false \
-      -Dxz=false \
-      -Dlz4=true \
-      -Dxkbcommon=false \
-      -Dpcre2=true \
-      -Dglib=false \
-      -Ddbus=false \
-      -Dgnu-efi=true \
-      -Duserdb=false \
-      -Dbashcompletiondir=no \
-      -Dzshcompletiondir=no \
-      -Dinstall-tests=false
-
-override_dh_auto_test:
-	LD_LIBRARY_PATH= LD_PRELOAD= dh_auto_test --buildsystem=meson+ninja --sourcedirectory=vendor/systemd || :
-
-override_dh_auto_clean:
-	dh_auto_clean --buildsystem=meson+ninja --sourcedirectory=vendor/systemd
-
-	sed -i '/"minix"/s/^\/*//' vendor/systemd/src/test/test-path-util.c
-
-	find vendor/systemd-patches -name "*.patch" -print0 |		\
-	    cut -z -d/ -f3 |						\
-	    sort -rz |							\
-	    (cd vendor/systemd;						\
-	     QUILT_PATCHES=debian/patches xargs -0 -n1 quilt delete -r)	\
-	  || true
-
-	cd vendor/systemd;					\
-	    QUILT_PATCHES=debian/patches quilt pop -a || true
-
-	rm -rf vendor/systemd/.pc
+	dh $@
 
+# bash is now used by debug-shell.service
 INSTALL_FILES_FROM_HOST=					\
+	/bin/bash						\
```

Comment:
I think we should override it in a drop-in instead.

Reply:
do we actually need full bash? previously, and initramfs-tools initrd we have been using ash only from our busybox compilation which is smaller.

Reply:
Removed now